src/Octoshift/RepositoryMigrationStatus.cs

```diff
@@ -6,5 +6,11 @@ public static class RepositoryMigrationStatus
         public const string InProgress = "IN_PROGRESS";
         public const string Failed = "FAILED";
         public const string Succeeded = "SUCCEEDED";
+        public const string PendingValidation = "PENDING_VALIDATION";
+        public const string FailedValidation = "FAILED_VALIDATION";
+
+        public static bool IsSucceeded(string migrationState) => migrationState?.Trim().ToUpper() is Succeeded;
+        public static bool IsPending(string migrationState) => migrationState?.Trim().ToUpper() is Queued or InProgress or PendingValidation;
+        public static bool IsFailed(string migrationState) => !(IsPending(migrationState) || IsSucceeded(migrationState));
```

Comment:
Not a huge deal but I guess it'd be better to add unit tests for these methods too, like the following:
```c#
public void IsPending_Returns_True_When_Status_Is_Queued() { ... }

public void IsPending_Returns_False_When_Statues_Is_Not_Queued_Or_InProgress_Or_PendindValidation() { ... }
```